MediaBrowser.Common/Net/NetworkExtensions.cs

```diff
@@ -45,7 +45,7 @@ public static void AddItem(this NetCollection source, IPObject item)
         /// </summary>
         /// <param name="source">The <see cref="NetCollection"/>.</param>
         /// <returns>Returns a string representation of this object.</returns>
-        public static string Readable(this NetCollection source)
+        public static string AsString(this NetCollection source)
         {
             var sb = new StringBuilder();
```

Comment:
Why create a StringBuilder if you don't use it?

Reply:
Good spot - left behind after an attempt to make a suggested change - and test. The overhead of using a sb with the number of iterations in this function was too big.